tests/test_pex_builder.py

```diff
@@ -190,6 +191,38 @@ def build_and_check(path, copy_mode):
         build_and_check(td4, CopyMode.SYMLINK)
 
 
+@pytest.fixture
+def tmp_chroot(tmpdir):
+    # type: (Any) -> Iterator[str]
+    tmp_chroot = str(tmpdir)
+    cwd = os.getcwd()
+    try:
+        os.chdir(tmp_chroot)
+        yield tmp_chroot
+    finally:
+        os.chdir(cwd)
+
+
+@pytest.mark.parametrize(
+    "copy_mode", [pytest.param(copy_mode, id=copy_mode.value) for copy_mode in CopyMode.values]
```

Comment:
Before the fix the test passed for `CopyMode.COPY` and `CopyMode.LINK` but failed for `CopyMode.SYMLINK`.